sycl/doc/design/CommandGraph.md

```diff
@@ -203,3 +203,42 @@ Level Zero:
 
 Future work will include exploring L0 API extensions to improve the mapping of
 UR command-buffer to L0 command-list.
+
+### OpenCL - cl_khr_command_buffer
+
+There are some gaps in both the OpenCL and UR specifications for Command Buffers shown in the list below. There are implementations in the UR OpenCL adapter where there is matching support for each function in the list.
```

Comment:
Can we link to the `cl_khr_command_buffer` spec in this intro

Reply:
Updated thanks.